mitosheet/mitosheet/tests/test_transpile.py

```diff
@@ -550,6 +550,39 @@ def test_transpile_as_function_single_param_multiple_times(tmp_path):
         f"txt, txt_1 = function(var_name)"
     ]
 
+def test_transpile_fully_parameterized_function_string(tmp_path):
+    tmp_file1 = str(tmp_path / 'txt.csv')
+    tmp_file2 = str(tmp_path / 'file.csv')
+    df1 = pd.DataFrame({'A': [1], 'B': [2]})
+    df1.to_csv(tmp_file1, index=False)
+    df1.to_csv(tmp_file2, index=False)
+
+    mito = create_mito_wrapper()
+    mito.simple_import([tmp_file1])
+    mito.simple_import([tmp_file2])
```

Comment:
deffo want more tests, namely one that tests all of the parameterization options.